src/_pytest/logging.py

```diff
@@ -297,6 +297,13 @@ def add_option_ini(option, dest, default=None, type=None, **kwargs):
         default=None,
         help="Auto-indent multiline messages passed to the logging module. Accepts true|on, false|off or an integer.",
     )
+    group.addoption(
+        "--suppress-logger",
```

Comment:
Have you considered my elaboration on the possible names for the CLI option on #7431 to start with a `--log...` prefix, to facilitate user's memory?